apps/scoutgame/middleware.ts

```diff
@@ -13,6 +13,10 @@ export async function middleware(request: NextRequest) {
   const response = NextResponse.next(); // Create a response object to set cookies
   const platform = getPlatform();
 
+  if (path === '/taiko') {
```

Comment:
good idea to put this here.. i woulda made a page.tsx file and everything, but this is cleaner